cypress/e2e/Testing/client-user.cy.js

```diff
@@ -0,0 +1,64 @@
+describe("User tests", () => {
+    it("CAN login and view their profile", () => {
+      let email = "Norofftest@noroff.no";
+      let password = "Norofftest@noroff.no";
+  
+      cy.visit("https://noroff-anders.github.io/Workflow-CA/");
+      cy.title().should("contain", "Test Client");
+      cy.wait(1000);
+  
+      cy.contains("Login").click({ force: true });
+      cy.wait(1000);
+  
+
+      cy.get("#loginEmail").type(email);
+      cy.get("#loginPassword").type(password);
+      cy.wait(1000);
+      cy.get("button.btn.btn-success").eq(1).click({force: true});
+      cy.wait(1000);
```

Comment:
There's a missing step, how do you prove that it worked?

Reply:
Added: 
``` cy.get("button[data-visible=loggedIn").should("exist"); ```
After logging in this checks for the data-visible=loggedIn property of the logout button exists